bindings/wasm/manifold-encapsulated-types.d.ts

```diff
@@ -111,6 +111,8 @@ export class CrossSection {
    * @param v The vector to add to every vertex.
    */
   translate(v: Vec2): CrossSection;
+  translate(x: number): CrossSection;
```

Comment:
Can we really take just an x-value? Feels weird...

Reply:
Oh, I guess this is how OpenSCAD does it, right?

Reply:
The functions already let you give a variable number of arguments, this just adds an overload to the typescript interface so you don't get res squigglies for legal use.

Reply:
> Oh, I guess this is how OpenSCAD does it, right?

Yeap that's right. We pretty well mirror that, since the `vararg2vec` helpers used are also flexible to array inputs not including all dimensions. Typescript will complain though, since that still isn't covered in the signature, e.g.: 
```js
translate(v: [number] | Vec2 | Vec3): Manifold
```